client/state/stats/lists/utils.js

```diff
@@ -189,8 +189,14 @@ export function getChartLabels( unit, date, localizedDate ) {
 		const isWeekend = 'day' === unit && ( 6 === dayOfWeek || 0 === dayOfWeek );
 		const labelName = `label${ unit.charAt( 0 ).toUpperCase() + unit.slice( 1 ) }`;
 		const formats = {
-			day: 'MMM D',
-			week: 'MMM D',
+			day: translate( 'MMM D', {
+				context: 'moment format string',
+				comment: 'This specifies a day for stats.',
+			} ),
+			week: translate( 'MMM D', {
+				context: 'moment format string',
```

Comment:
Have we done this elsewhere? I'm curious if a translator will know how to interpret `moment format string`.

Reply:
Cool, thanks. The example is translating the "today" part of the string only. In this case, the translation would be the format itself. In German, (I believe) would be "D. MMM".

```js
moment().locale()
// "de"

moment().format('D. MMM');
// "13. Juni"
```
If the translators are Ok with with this, then 🚢 from me

Reply:
Yeah, translators might also decide something like `[Week] W`to display the week number, having the format translatable allows a lot of flexibility.